server/routeHandlers.go

```diff
@@ -53,103 +53,107 @@ func newGame(c echo.Context) error {
 }
 
 func login(c echo.Context) error {
-    player, joinErr := joinGame(c.Param("game"), c.Param("username"))
-    
-    if joinErr != nil {
+	isHost := false
+	if c.Param("isHost") == "true" {
+		isHost = true
+	}
+	player, joinErr := joinGame(c.Param("game"), c.Param("username"))
+
+	if joinErr != nil {
 		return joinErr
 	}
-    
-    return respondWithJWTIfValid(c, player, joinErr == nil)
+
+	return respondWithJWTIfValid(c, player, joinErr == nil, isHost)
 }
 
 func startGame(c echo.Context) error {
-    
-    authHeader := c.Request().Header.Get(echo.HeaderAuthorization)
-    
-    claims, validUser := getValidClaimsFromHeader(authHeader)
+	authHeader := c.Request().Header.Get(echo.HeaderAuthorization)
+
+	claims, validUser := getValidClaimsFromHeader(authHeader)
 
 	if !validUser || claims["isHost"] != true {
 		return c.JSONPretty(http.StatusUnauthorized, &Response{false, nil}, " ")
 	}
-    
+
 	dealCards()
 	return update(c)
 }
 
 func update(c echo.Context) error {
-    
-    authHeader := c.Request().Header.Get(echo.HeaderAuthorization)
-    
-    claims, validUser := getValidClaimsFromHeader(authHeader)
+
+	authHeader := c.Request().Header.Get(echo.HeaderAuthorization)
+
+	claims, validUser := getValidClaimsFromHeader(authHeader)
 
 	if !validUser {
 		return c.JSONPretty(http.StatusUnauthorized, &Response{false, nil}, " ")
 	}
 
 	valid := updateGame(claims["gameid"].(string))
-	return respondIfValid(c, valid && validUser, claims["userid"].(string), claims["gameid"].(string))
+	return respondIfValid(c, valid && validUser, claims["name"].(string), claims["gameid"].(string))
 }
 
 func play(c echo.Context) error {
-    
-    authHeader := c.Request().Header.Get(echo.HeaderAuthorization)
-    
-    claims, validUser := getValidClaimsFromHeader(authHeader)
+	authHeader := c.Request().Header.Get(echo.HeaderAuthorization)
+
+	claims, validUser := getValidClaimsFromHeader(authHeader)
 
 	if !validUser {
 		return c.JSONPretty(http.StatusUnauthorized, &Response{false, nil}, " ")
 	}
-    
-    // TODO Cards have a value, which can include skip, reverse, etc
-    card := model.Card{c.Param("number"), c.Param("color")}
-	valid := playCard(claims["gameid"].(string), claims["userid"].(string), card)
-	return respondIfValid(c, valid, claims["userid"].(string), claims["gameid"].(string))
+
+	// TODO Cards have a value, which can include skip, reverse, etc
+	card := model.Card{c.Param("number"), c.Param("color")}
+	valid := playCard(claims["gameid"].(string), claims["name"].(string), card)
+	return respondIfValid(c, valid, claims["name"].(string), claims["gameid"].(string))
```

Comment:
This would make it so that people who have the same name get mixed up (i.e. if our game had two people with the name "John"). We need to use the userid, as it is the only value guaranteed to be unique to the player.